content/en/blog/2025/otel-weaver.md

```diff
@@ -0,0 +1,329 @@
+---
+title: Observability by Design: Unlocking Consistency with OpenTelemetry Weaver
+linkTitle: OTel Weaver
+date: 2025-06-27
+author: [Laurent Quérel](https://github.com/lquerel), [Jeremy Blythe](https://github.com/jerbly), [Josh Suereth](https://github.com/jsuereth), [Liudmila Molkova](https://github.com/lmolkova)
+sig: "Semantic Conventions: Tooling"
+cSpell:ignore: Liudmila Molkova promconv Quérel SDLC Suereth
+---
+
+{{% alert title="TL;DR" %}} Stop treating observability as an afterthought.
+Start building it in by design, with clear conventions, type safety, and
+automation. {{% /alert %}}
+
+## Have you ever experienced…?
+
+- A deployment that breaks existing alerts or dashboards because a metric name
+  changed?
+- Writing overly complex queries because teams use different metric names for
+  the same thing?
+- Losing hours debugging production issues due to missing or unclear
+  instrumentation?
+- Teams struggling to interpret undocumented or inconsistent metrics?
+
+If any of this sounds familiar, you're not alone. These are the symptoms of
+treating telemetry as an afterthought, instead of an intentional part of your
+software design.
+
+## Semantic Conventions: The Foundation for Consistent Observability
+
+**Semantic conventions** are a set of rules and standard names for telemetry
+data. Think of them as the "grammar" for metrics, traces, and logs, so everyone
+and everything (including your tools) knows what you mean by
+'http.request.method', 'db.system.name', or 'http.client.request.duration'.
+
+The
+[OpenTelemetry Semantic Convention](https://opentelemetry.io/docs/specs/semconv/)
+is a massive, open catalog with 900+ attributes and signals in over 70
+domains-maintained by nine special interest groups. This open catalog ensures:
+
+- **Consistency**: One name, one meaning everywhere.
+- **Interoperability**: Tools, teams, and vendors can understand each other.
+- **Automation**: Machine-readable standards enable code and docs generation,
+  static and live compliance checks, and more.
+
+But maintaining and evolving such a registry across teams and tools isn't easy.
+That's where [OTel Weaver](https://github.com/open-telemetry/weaver) comes in.
```

Comment:
I like this section very much, and would prefer it as an introduction to the blog post.

Reply:
I did a sort of merge between these two sections and redefined its title. I like the question format because many people will relate to it, and I use it to better introduce the concept of semantic conventions.